gpjax/kernels/stationary/changepoint.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+"""
+AUTHOR: Jake Yeung (jakeyeung@gmail.com)
+CREATED ON:  2023-07-28
+LAST CHANGE: see git log
+LICENSE:     Apache License
+"""
+
+from dataclasses import dataclass
+
+from beartype.typing import Union
+import jax
+import jax.numpy as jnp
+from jaxtyping import Float
+import tensorflow_probability.substrates.jax.bijectors as tfb
+
+import gpjax as gpx
+from gpjax.base.param import param_field
+from gpjax.typing import (
+    Array,
+    ScalarFloat,
+)
+
+
+@dataclass
+class ChangePointRBF(gpx.kernels.AbstractKernel):
+    r"""The change point kernel for two RBFs."""
+
+    variance1: ScalarFloat = param_field(
+        jnp.array(1.0), bijector=tfb.Softplus(), trainable=True
+    )
+    lengthscale1: Union[ScalarFloat, Float[Array, " D"]] = param_field(
+        jnp.array(1.0), bijector=tfb.Softplus(), trainable=True
+    )
+    variance2: ScalarFloat = param_field(
+        jnp.array(1.0), bijector=tfb.Softplus(), trainable=True
+    )
+    lengthscale2: Union[ScalarFloat, Float[Array, " D"]] = param_field(
+        jnp.array(1.0), bijector=tfb.Softplus(), trainable=True
+    )
+    tswitch: float = param_field(
```

Comment:
It would be good to acknowledge what this is in the docstring.

Reply:
I added what tswitch does in the initialization of the `SwitchKernel`